.github/workflows/ci-tests-n-coverage.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           cargo llvm-cov report --lcov --output-path ./lcov.info
 
       - name: Upload code coverage to coveralls.io
-        uses: coverallsapp/github-action@master
+        uses: coverallsapp/github-action@tests/typhon-registration
```

Comment:
You should not change this because it refers to the master branch of this repo coverallsapp/github-action